### PR TITLE
Update directions to work :)

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

**Comment by jayunit100** [comment]
cc @jeremyeder @portante

File: pbench-agent.html [hunk 1]
```diff
@@ -169,8 +169,9 @@ <h2 id="sec-4"><span class="section-number-2">4</span> TL;DR version</h2>
 	      Here is the minimum set of commands to get from 0 to a crawl:
 	    </p>
 	    <pre><code>
-        wget -O /etc/yum.repos.d/pbench.repo http://repohost.example.com/repo/yum.repos.d/pbench.repo
-        yum install pbench-agent -y
+	sudo wget -O /etc/yum.repos.d/pbench.repo https://raw.githubusercontent.com/redhat-performance/openshift-performance/master/repos/pbench.repo
```
Comment (portante) [inline pbench-agent.html]: Is this pbench repo file usable for the general pbench user?

Comment (jayunit100) [inline pbench-agent.html]: Works better than "example.com/fake.repo" :)

Comment (jeremyeder) [inline pbench-agent.html]: While it works,  I used that for a demo and it shouldn't be part of anyone's workflow.  There needs to be a single repo file hosted within the pbench repo itself.  I wonder if the ansible jobs we have should become the public "pbench installer".

Comment (jayunit100) [inline pbench-agent.html]: can we maybe merge with intent to improve?  right now what is in the instructions is literally completely useless

Comment (jeremyeder) [inline pbench-agent.html]: copr generates repo files here:  https://copr.fedorainfracloud.org/coprs/ndokos/pbench/

File: pbench-agent.html [hunk 2]
```diff
@@ -169,7 +169,7 @@ <h2 id="sec-4"><span class="section-number-2">4</span> TL;DR version</h2>
 	      Here is the minimum set of commands to get from 0 to a crawl:
 	    </p>
 	    <pre><code>
-	sudo wget -O /etc/yum.repos.d/pbench.repo https://raw.githubusercontent.com/redhat-performance/openshift-performance/master/repos/pbench.repo
+	sudo wget -O /etc/yum.repos.d/pbench.repo https://copr.fedorainfracloud.org/coprs/ndokos/pbench/repo/epel-7/ndokos-pbench-epel-7.repo
```
Comment (jayunit100) [inline pbench-agent.html]: @jeremyeder ok, this appears to work for me thanks!

**Comment by ndokos** [comment]
There is an extensive doc revision in PR #266, which addresses this point (among many others). PR #267 then updates the README.md file in the master branch to point to the revised docs. 

Closing this PR.

**Comment by jayunit100** [comment]
Fair enough thanks!

**Pull request closed without merging**